THANKS.md

```diff
@@ -1,3 +1,15 @@
+# The rav1d project would like to thank
+
+## Prossimo
+The Internet Safety Research Group and its donors for funding this Rust port.
+
+## Contributors
+
+The rav1d Rust port authors:
+
+Khyber Sen, Nicole L, Per Larsen, Stephen Crane, Folkert de Vries
```

Comment:
Should we also list @fbossen here?

Reply:
Oh my! Yes, so sorry, I went through the top contributors and missed Frank!